modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/AuditBarChart.js

```diff
@@ -183,7 +183,11 @@ export default function AuditBarChart({rtl, vocabularies}) {
 
 	const axisNames = {
 		x: vocabularies[0]?.vocabularyName,
-		y: showLegend && vocabularies[0]?.categories?.[0]?.vocabularyName,
+		y:
+			showLegend &&
+			vocabularies.find(
+				(vocabulary) => vocabulary.categories !== undefined
```

Comment:
You could probably simplify this to:

```js
vocabularies.find(({categories}) => categories)
```

instead of explicitly comparing against `undefined`.

Reply:
You're right! 😄  I'll update the changes.